client/src/components/pages/meditation.tsx

```diff
@@ -1,168 +1,159 @@
+    ],
+    "believe-in-yourself": [
+      { title: "Focus on Your Strengths", content: "Reflect on your accomplishments and strengths. Picture yourself succeeding in your goals." },
+      { title: "Let Go of Doubt", content: "Visualize letting go of any doubts or insecurities as though they are drifting away in the wind." },
+      { title: "Embrace Your Potential", content: "See a version of yourself who is unstoppable, full of confidence, and belief in your abilities." },
+      { title: "Stay Positive", content: "In every challenge, find the opportunity for growth. Focus on the positive in each situation." },
+      { title: "Final Affirmation", content: "You are strong, capable, and worthy of success. Keep believing in yourself." },
+    ],
+    "time-to-sleep": [
+      { title: "Prepare for Sleep", content: "Dim the lights and lie comfortably in bed. Let your body relax and unwind." },
+      { title: "Release Tension", content: "Starting from your feet, release any tension. Focus on each body part, and let it relax completely." },
+      { title: "Breathe Deeply", content: "Take deep, slow breaths in and out. With each exhale, feel yourself sinking deeper into relaxation." },
+      { title: "Let Go of the Day", content: "Visualize your thoughts from the day floating away like clouds, leaving you in peaceful stillness." },
+      { title: "Sleep Well", content: "As you drift off, imagine yourself in a place of peace and tranquility. Sleep comes easily to you." },
+    ],
+    "mindful-breathing": [
+      { title: "Focus on Your Breath", content: "Take a deep breath in and exhale slowly. Feel the air enter and leave your body." },
+      { title: "Breathe Mindfully", content: "With each breath, focus only on the sensation of breathing. Let go of all other thoughts." },
+      { title: "Notice the Calm", content: "As you breathe, notice the sense of calmness settling over you with each inhale and exhale." },
+      { title: "Be Present", content: "Inhale deeply, and exhale fully. Allow yourself to be fully present in this moment." },
+      { title: "Feel Peaceful", content: "With every breath, feel a growing sense of peace filling your body and mind." },
+    ],
+    "positive-energy-boost": [
+      { title: "Feel the Energy", content: "Take a deep breath and imagine a bright, vibrant energy filling your body." },
+      { title: "Activate Your Mind", content: "Visualize your mind being filled with positive thoughts and energy." },
+      { title: "Energize Your Body", content: "Focus on each body part, feeling a surge of energy revitalizing you." },
+      { title: "Embrace Positivity", content: "Visualize a light surrounding you, pushing out negativity and replacing it with vibrant positivity." },
+      { title: "Feel Empowered", content: "You are full of energy and confidence. Embrace the positivity within you." },
```

Comment:
The text is very hard to see here, maybe change the color

![image](https://github.com/user-attachments/assets/831ac56e-9cd1-4b13-9cf3-bea1715eed86)

Reply:
fixed


<img width="1046" alt="Screenshot 2024-12-04 at 8 29 41 AM" src="https://github.com/user-attachments/assets/3c0b580a-64c1-4109-bab2-72da36d68612">